server/services/prismic.js

```diff
@@ -157,8 +156,9 @@ export async function getArticleSeries(seriesId) {
   }, {items: List(scheduleItems)}, {id: seriesId});
 }
 
-export async function getSeriesAndArticles(id: string, page: number = 1) {
-  const paginatedResults = await getArticleList(page, {
+export async function getSeriesAndArticles(cookies, {id, page = 1} = {}) {
```

Comment:
Basically this will be the shape of all Prismic getter functions.
As per a discussion with @gestchild 
```flow
type  PrismicGetter<T, O> = (cookies: Cookies, options: O) => T;
const getArticle: PrismicGetter<Article, { id: string }> = (cookies, options) => {
  const article = prismicJazz;
  return article;
}
```

Reply:
[e.g](https://flow.org/try/#0C4TwDgpgBAwg9nA1gSwgZygXigbwL4DcAUKJFAIIBOwyAxgDbTY4A+UyAJgFxRrCXIAdgHMCUGsEY8+AkVBaES4aAAUBaALZ0A4hGDAIlADwAVADRQA8gD4sUABS0EKdD3hJUaC3DA04gtB5LAEosWxNiIicA4ChhPSoaBggeNWRNHT0DY0S6Rgscdm5efiFhKDxbbEdnT29fZH80UMxbHCIoKGi+KABDajymXCKeHz8AgDpOCwkpKAByAAkIeno4cQALQwh5iuJOyj0AV0pBPoHk4jwiKKbY-uAeXOS7eOBnxnt8As4eeY4EBx6CAAXB5nhggQgA)